src/Ens/JobeetBundle/Controller/JobController.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace Ens\JobeetBundle\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+use Ens\JobeetBundle\Entity\Job;
+use Ens\JobeetBundle\Form\JobType;
+
+/**
+ * Job controller.
+ *
+ */
+class JobController extends Controller
+{
+
+    /**
+     * Lists all Job entities.
+     *
+     */
+    public function indexAction()
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $entities = $em->getRepository('EnsJobeetBundle:Job')->findAll();
+
+        return $this->render('EnsJobeetBundle:Job:index.html.twig', array(
+            'entities' => $entities,
+        ));
+    }
+    /**
+     * Creates a new Job entity.
+     *
+     */
+    public function createAction(Request $request)
+    {
+        $entity = new Job();
+        $form = $this->createCreateForm($entity);
+        $form->handleRequest($request);
+
+        if ($form->isValid()) {
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($entity);
+            $em->flush();
+
+            return $this->redirect($this->generateUrl('ens_job_show', array('id' => $entity->getId())));
+        }
+
+        return $this->render('EnsJobeetBundle:Job:new.html.twig', array(
+            'entity' => $entity,
+            'form'   => $form->createView(),
+        ));
+    }
+
+    /**
+     * Creates a form to create a Job entity.
+     *
+     * @param Job $entity The entity
+     *
+     * @return \Symfony\Component\Form\Form The form
+     */
+    private function createCreateForm(Job $entity)
+    {
+        $form = $this->createForm(new JobType(), $entity, array(
+            'action' => $this->generateUrl('ens_job_create'),
+            'method' => 'POST',
+        ));
+
+        $form->add('submit', 'submit', array('label' => 'Create'));
+
+        return $form;
+    }
+
+    /**
+     * Displays a form to create a new Job entity.
+     *
+     */
+    public function newAction()
+    {
+        $entity = new Job();
+        $form   = $this->createCreateForm($entity);
+
+        return $this->render('EnsJobeetBundle:Job:new.html.twig', array(
+            'entity' => $entity,
+            'form'   => $form->createView(),
+        ));
+    }
+
+    /**
+     * Finds and displays a Job entity.
+     *
+     */
+    public function showAction($id)
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $entity = $em->getRepository('EnsJobeetBundle:Job')->find($id);
+
+        if (!$entity) {
+            throw $this->createNotFoundException('Unable to find Job entity.');
```

Comment:
@PhotonSapiens 
you can omit this code block by using `@ParamConverter` annotation: 
http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/converters.html